cms/core/blocks/related.py

```diff
@@ -31,6 +31,30 @@ def get_context(self, value: "ListValue", parent_context: dict | None = None) ->
         context["heading"] = self.heading
         context["slug"] = self.slug
 
+        related_links = []
+
+        for item in value:
+            url = item.get("external_url")
+            if not url and item.get("page"):
+                url = item["page"].url
+
+            if not url:
+                continue
+
+            text = item.get("title")
+            if not text:
+                continue
+
+            link_data = {"title": {"text": text, "url": url}}
+
+            description = item.get("description")
+            if description:
+                link_data["description"] = description
+
+            related_links.append(link_data)
+
+        context["related_links"] = related_links
```

Comment:
`RelatedLinksBlock` is a list of `RelatedContentBlock` (which extends [`LinkBlock`](https://github.com/ONSdigital/dis-wagtail/blob/1dec7c3740c126136689ed7bb4c3347ad39f462c/cms/core/blocks/base.py#L40) that in turn has a helpful [`LinkBlockStructValue`](https://github.com/ONSdigital/dis-wagtail/blob/1dec7c3740c126136689ed7bb4c3347ad39f462c/cms/core/blocks/base.py#L14) which provied a `link` property that currently outputs data like `{'url': '/', 'text': 'The title', 'description': ''}`.

I suggest updating that to `{"title": {'url': the_relevant_url, 'text': the_title}` and so this then becomes:

context["related_links"] = [item.link for item in value]

Reply:
If we update LinkBlockStructValue to output a different structure for this, is that going to have an impact on any other blocks using it? Or is that not a problem? apologies if it's an obvious q!